logos/logos-ui/src/app/features/model-error-report/model-error-report.scss

```
@@ -101,39 +106,217 @@
+  text-align: center;
+}
+
+.performance-list {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(420px, 1fr));
+  gap: 20px;
+}
+
+.performance-card {
+  @include glass-surface;
+  padding: 24px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)